docs/dev/tutorials/api3-usd-paymaster-tutorial.md

```diff
@@ -87,12 +99,15 @@ contract MyERC20 is ERC20 {
 }
 ```
 
-Under contracts, you will find `Greeter.sol`. This is the contract that we will be using to test our paymaster to set a greeting message on-chain.
+The `Greeter.sol` contract is already in the `contracts` directory. This is the contract that we will be using to test our paymaster to set a greeting message on-chain.
 
-### Paymaster solidity contract
+### 2. Paymaster solidity contract
 
+Create `MyPaymaster.sol` in the `/contracts` directory. 
 
-2. We can now create our paymaster contract `MyPaymaster.sol` under `/contracts` directory. As it is a custom implementation of the [zkSync paymaster](/docs/dev/tutorials/custom-paymaster-tutorial.html) contract that uses dAPIs, make sure you've followed the guide to understand the basics of paymasters.
+:::tip Tip
+As it is a custom implementation of the zkSync Era paymaster contract that uses dAPIs, make sure you follow the [zkSync paymaster](/docs/dev/tutorials/custom-paymaster-tutorial.html) to understand the basics of paymasters.
+:::
```

Comment:
I'd delete this tip. It points to the other paymaster tutorial but it adds no value in my opinion

Reply:
let's do it